react-components/src/components/Architecture/SettingsButton.tsx

```diff
@@ -266,74 +281,78 @@ function createSlider(command: BaseSliderCommand, t: TranslateDelegate): ReactNo
   );
 }
 
-function createDropdownButton(command: BaseOptionCommand): ReactNode {
+function DropdownButtonComponent({ command }: { command: BaseOptionCommand }): ReactNode {
   // @update-ui-component-pattern
   const [isVisible, setVisible] = useState(true);
-  const [uniqueId, setUniqueId] = useState(0);
 
   useOnUpdate(command, () => {
     setVisible(command.isVisible);
-    setUniqueId(command.uniqueId);
   });
   // @end
 
   if (!isVisible) {
     return null;
   }
 
-  return (
-    <DropdownButton
-      key={uniqueId}
```

Comment:
What happened to the `key`s now? Are we passing these while using `DropdownButtonComponent` now?